src/modules/site-v2/base/views/strains.py

```diff
@@ -324,7 +327,8 @@ def order_page_index():
     'title': "Order Summary",
     'cartItems': cartItems,
     'totalPrice': totalPrice,
-    'form': form
+    'form': form,
+    'EULA_url': get_blob(MODULE_SITE_BUCKET_PRIVATE_NAME, 'EULA_for_CaeNDR_website_Dr_Andersen_1-25-24.md').public_url

```

Comment:
we need to normalize all json payloads format -- consider using lowercase with underscores (🐍 case)